### ISSUE TITLE
Instructions or data?

### ISSUE THREAD

**Issue by jayurbain** [JakeRenn/pose_estimation#1]
Hi,

I came across your repository and it looks like an interesting implementation of tensorflow for pose estimation.  Do you have any additional instructions for using your software? E.g., what input data format, etc.

Thanks,
Jay
jay.urbain@gmail.com

**Comment by JakeRenn** [comment]
@jayurbain Hi.

This is a simplified version of my project, which is used as a demo for my friends. Of course, this project is part of my main projects, but lots of details and more efficient functions are still unpublished. 

I will release the complete version of this project when my paper is accepted. And more information will be provided then, including the instruction for using my software and how to prepare the input data format.

**Comment by jayurbain** [comment]
Ok. Thanks.

**Comment by Bigwode** [comment]
Hello, Thanks for your project first ! But when I looked through your code and I had no idea of how to transform the original .mat annotations to .txt as the training data's labels to suit your project. So, can you share your skills in how to do this work or share your training data's labels by your convenient.Thanks a lot !
